*** note	Mon Jul 10 00:10:03 2000
--- _note	Fri Jul 21 08:38:22 2000
***************
*** 635,641 ****
  	    	$editor = &find_editor;
          	if($editor)
          	{
! 		        system "touch", $TEMP && die $!;
  			system "chattr", "+s", $TEMP; # ignore errors, since only on ext2 supported!
          	        system $editor, $TEMP;
          	}
--- 635,641 ----
  	    	$editor = &find_editor;
          	if($editor)
          	{
! 		        system "touch", $TEMP and die $!;
  			system "chattr", "+s", $TEMP; # ignore errors, since only on ext2 supported!
          	        system $editor, $TEMP;
          	}
